lark/exceptions.py

```diff
@@ -193,7 +191,7 @@ def __init__(self, token, expected, considered_rules=None, state=None, interacti
         # TODO considered_rules and expected can be figured out using state
         self.line = getattr(token, 'line', '?')
         self.column = getattr(token, 'column', '?')
-        self.pos_in_stream = getattr(token, 'pos_in_stream', None)
+        self.pos_in_stream = getattr(token, 'start_pos', None)
```

Comment:
We can just rename the attribute.

Reply:
You mean, in the exceptions?

I'm less concerned about that (and anyway, `start_pos` wouldn't be a good name in this case). But I'm open to suggestions.

Reply:
Fair enough. Keep `pos_in_stream`